src/components/Category/CategoryCard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="card category-card">
-    <img :src="category.imageUrl" class="card-img-top" alt="image" @click="listProducts">
+    <img :src="this.imageUrl" class="card-img-top" alt="image" @click="listProducts">
```

Comment:
Why are you using this instead of category?

Reply:
I am binding the data ... 
data(){
    return{
      imageUrl:''
    }
  },
  mounted(){
    this.imageUrl = this.category.imageUrl
  }